docs/bugs.json

```diff
@@ -1,14 +1,22 @@
 [
     {
+        "name": "DelegateCallReturnValue",
+        "summary": "The low-level .delegatecall() took the returned value as a boolean instead of the execution outcome.",
```

Comment:
I think the other summaries are written in present tense.
Also note that the returned value can be garbage, if the called contract does not return anything.